mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -666,6 +636,27 @@
     <!-- Preference for site permissions -->
     <string name="preference_site_permissions">Site permissions</string>
 
+    <!--  Preference for cookie banner. The name of the item from settings screen and the title of the cookie banner screen. -->
+    <string name="preferences_cookie_banner">Cookie Banner Reduction</string>
```

Comment:
What is Cookie Banner? Please give more context to localizers in comment (a link to a screenshot would be even better)